SConstruct

```diff
@@ -212,7 +212,6 @@ config_options = [
            optimization level.""",
         {
             "cl": "/MD /nologo /D_SCL_SECURE_NO_WARNINGS /D_CRT_SECURE_NO_WARNINGS",
-            "icc": "-vec-report0 -diag-disable 1478",
```

Comment:
I'd prefer if we didn't drop the `icc` stuff, especially since folks are likely to keep using it for some time to come, even though it's been deprecated.

Reply:
I added the runner back in ... as you can see in https://github.com/Cantera/cantera/runs/4460844601?check_suite_focus=true there aren't any issues